apps/judicial-system/backend/src/app/modules/case/limitedAccessCase.controller.ts

```diff
@@ -165,6 +167,15 @@ export class LimitedAccessCaseController {
       update.accusedPostponedAppealDate = nowFactory()
     }
 
+    if (transition.transition === CaseTransition.WITHDRAW_APPEAL) {
+      if (
+        !theCase.appealRulingDecision &&
+        theCase.appealState === CaseAppealState.RECEIVED
+      ) {
```

Comment:
Please explain why this condition identifies a case that has not been received by the court of appeals.

Based on the changes to `case.filter.ts` and `cases.filter.ts` I would think you can simple skip the `appealRulingDecision` check.

Reply:
Same answer as above